_posts/elements/2015-04-01-colors.md

```diff
@@ -16,8 +16,8 @@ description:  Pattern Library color swatches
 info: These are the predefined colors used throughout our patterns and elements. Values here (noted in RGB format as well by the relative scale name) are used for backgrounds, text color, and decorative elements. Note - when using any values here, make sure that any text on a background passes WCAG AA guidelines for color contrast.
 ---
 
+<h3 class="hd-6 swatch-set-hd">Brand Primary</h3>
```

Comment:
NOTE: had to move these headings out of the nested ``swatch-set`` since Neat was counting them in its nth-child grid rules.